### ISSUE TITLE
First SSP 100M Training Runs Kicked Off

### ISSUE THREAD

**Issue by Plyb** [DRAGNLabs/MIRROR-Pipeline#252]
**Acceptance Criteria**:
- A training run for a 100M parameter model has been kicked off, and seems to be running successfully for the following datasets:
  - The SSP Solace data
  - The SSP BYU data
  - The SSP Internet data
  - A `MixedDataset` combining the above 3
- Each training run is specified with a very large number of epochs so that we can train to validation-loss convergence
  - if reasonable, we should also train to weight-norm convergence\\

Misc thoughts on efficient training:
- Disable cross-document attention (see last paragraph of section 2 http://arxiv.org/abs/2501.01956)
- Possible source of hyperparams: Datacomp-lm: In search of the next generation of training sets for language models. arXiv preprint arXiv:2406.11794, 2024.
